app-backup/rear/rear-2.4.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Relax-and-Recover is a setup-and-forget bare metal disaster recovery solution"
+HOMEPAGE="http://relax-and-recover.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Are you sure about that? I've just looked at [random file](https://github.com/rear/rear/blob/master/usr/sbin/rear) and it says GPL-3+.

Reply:
I changed the License.

Reply:
Why `GPL-3` and not `GPL-3+`?

Reply:
My mistake sorry, i fixed it.